### DISCUSSION TITLE
Removing the `data` key in transformers

### DISCUSSION THREAD

**Discussion by innoflash** [apiato/apiato#738] [General]
```
{
    "data": {
        "object": "Profile",
        "id": "z3RboB8zn87peM2x",
        "age": 84,
        "gender": "Male",
        "build": "occaecati",
        "hairColor": "DarkBlue",
        "height": "qui",
        "eyeColor": "DarkBlue",
        "hasBodyArt": false,
        "user": {
            "data": {
                "object": "User",
                "id": "z3RboB8zn87peM2x",
                "name": "Christop Kozey",
                "email": "hwisoky@example.org",
                "phone": "(458) 751-1803",
                "created_at": "2024-01-17T11:27:46.000000Z"
            }
        }
    },
    "meta": {
        "include": [
            "user"
        ],
        "custom": []
    }
}
```

Hey guys, just wondering if there is a way to take out the data key from my responses.

How does one do that if there is a way?

**Comment by Mohammad-Alavi** [comment]
You can change the default serializer in `app/Ship/Configs/fractal.php` and use your own serializer or extent one of the existing ones and do whatever you like 💪

**Reply by innoflash** [reply]
In short there is no easy way to do this? :)

Would you have a link or demo code on how to override this practically.

the `\Spatie\Fractal\Fractal::class` used there is not an easy read.

Wou you have like a link a to a documentation on how I can make an override?

Cheers

**Reply by Mohammad-Alavi** [reply]
You can use [ArraySerializer](https://fractal.thephpleague.com/serializers).

**Reply by innoflash** [reply]
Thanks @Mohammad-Alavi 

Definitely worked, even found a way on how to customize it to my needs :)

**Reply by Mohammad-Alavi** [reply]
For anyone else reading this, Apiato [docs](https://apiato.io/docs/components/main-components/transformers#response-payload) already cover this.
